lib/themes/dosomething/paraneue_dosomething/js/campaign/prototype.js

```diff
@@ -25,6 +27,17 @@ function makeKudosRequest($el, id)
 }
 
 /**
+ * Make an API request to Rogue to create/delete the Reaction for that item.
+ *
+ * @param string User's northstar ID.
+ */
+ function makeKudosRequestToRogue($northstarId, $postId)
```

Comment:
Let's use the same Phoenix reactions endpoint for now so the front-end doesn't need to change!